app/models/developer_external_profiles/linkedin_profile_fetcher.rb

```diff
@@ -0,0 +1,48 @@
+module DeveloperExternalProfiles
+  class LinkedinProfileFetcher
+    def developer_profiles
+      developer_external_profiles_list = []
```

Comment:
`profiles`, maybe? There's a lot of text in this method making it a bit hard to read.

Reply:
And maybe rename `record` a few lines down.